### PR TITLE
Add Veo line graph and switching types

### PR DESCRIPTION
**Pull request by ischolten**
Connects https://github.com/influxdata/platform/issues/1038

_Briefly describe your proposed changes:_
You can now render a visualization with dummy data in the CEO and DE. 
Clicking on the Visualizations tab will show a graph type selector. 
Only dygraph graph types will work right now.

  - [x] Rebased/mergeable
  - [x] Tests pass

### CODE REVIEW

File: chronograf/ui/src/shared/actions/v2/timeMachines.ts [hunk 1]
```diff
@@ -36,3 +37,13 @@ export const setTimeRange = (timeRange: TimeRange): SetTimeRangeAction => ({
   type: 'SET_TIME_RANGE',
   payload: {timeRange},
 })
+
+interface SetTypeAction {
+  type: 'SET_VIEW_TYPE'
+  payload: {type: ViewType}
+}
+
+export const setType = (type: ViewType): SetTypeAction => ({
```
Comment (OfTheDelmer) [inline chronograf/ui/src/shared/actions/v2/timeMachines.ts]: Nice

File: chronograf/ui/src/shared/components/threesizer/Division.tsx [hunk 2]
```diff
@@ -70,18 +72,25 @@ class Division extends PureComponent<Props> {
...
+
   public render() {
     const {render} = this.props
+
     return (
       <div
         className={this.containerClass}
         style={this.containerStyle}
-        ref={this.ref}
+        ref={this.divisionRef}
```
Comment (OfTheDelmer) [inline chronograf/ui/src/shared/components/threesizer/Division.tsx]: nice rename

File: chronograf/ui/src/shared/reducers/v2/timeMachines.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // Utils
-import {getNewView} from 'src/dashboards/utils/cellGetters'
+// import {getNewView} from 'src/dashboards/utils/cellGetters'
```
Comment (OfTheDelmer) [inline chronograf/ui/src/shared/reducers/v2/timeMachines.ts]: Could we delete this now?

Comment (ischolten) [reply chronograf/ui/src/shared/reducers/v2/timeMachines.ts]: it'll be used once we are no longer hard coding the values but i can delete it now

**Pull request merged**